MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.AppCompat/MvxAppCompatViewPresenter.cs

```diff
@@ -487,6 +487,20 @@ protected virtual bool CloseViewPagerFragment(IMvxViewModel viewModel, MvxViewPa
             return false;
         }
 
+        protected override bool CloseFragments()
+        {
+            try
+            {
+                CurrentFragmentManager.PopBackStackImmediate();
```

Comment:
Can we run this code only if there are entries on the backstack instead of running it inside of a try/catch block?

Reply:
This is too prevent AppCompat and normal fragment type exceptions.

Reply:
Alright, I think that makes sense as long as we print the exception. Does it make sense to call base in this context, when base uses a different fragment manager @martijn00? 
Sounds like in all cases two exceptions will be written in the output because of that.

Reply:
That is exactly why i call base, because base has a different manager which might be the correct type.

Reply:
Sorry for insisting but... When using the AppCompatViewPresenter there's no way to show a non-compat Fragment, [ShowFragment](https://github.com/martijn00/MvvmCross/blob/025dfabe03308c270661a7b6affe3b108334a5d3/MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.AppCompat/MvxAppCompatViewPresenter.cs#L201) does not call base in any case. 

I think there are no scenarios where the base FragmentManager could resolve the request

Reply:
Hmm yeah you are right with that. I would still leave the try catch but remove the base call.

Reply:
Yes, good with me 👍